_episodes/06-raster-intro.md

```diff
@@ -0,0 +1,462 @@
+{: .callout}
+
+## Visualize a Raster
+
+After viewing the attributes of our raster, we can examine the raw values of the array with `.values`:
+
+~~~
+raster_ams_b9.values
+~~~
+{: .language-python}
+~~~
+array([[[    0,     0,     0, ...,  8888,  9075,  8139],
+        [    0,     0,     0, ..., 10444, 10358,  8669],
+        [    0,     0,     0, ..., 10346, 10659,  9168],
+        ...,
+        [    0,     0,     0, ...,  4295,  4289,  4320],
+        [    0,     0,     0, ...,  4291,  4269,  4179],
+        [    0,     0,     0, ...,  3944,  3503,  3862]]], dtype=uint16)
+~~~
+{: .output}
+
+This can give us a quick view of the values of our array, but only at the corners. Since our raster is loaded in python as a `DataArray` type, we can plot this in one line similar to a pandas `DataFrame` with `DataArray.plot()`.
+
+~~~
+raster_ams_b9.plot()
+~~~
+{: .language-python}
+
+<img src="../fig/E06-01-overview-plot-B09.png" title="Raster plot with rioxarray" alt="Raster plot with defualt setting" width="612" style="display: block; margin: auto;" />
+
+Nice plot! Notice that `rioxarray` helpfully allows us to plot this raster with spatial coordinates on the x and y axis (this is not the default in many cases with other functions or libraries). 
+
+This map shows the measurement from the spectral band `B09`. According to the [documentaion](https://custom-scripts.sentinel-hub.com/custom-scripts/sentinel-2/bands/), this is a band with the central wavelength of 945nm, which is sensitive to water vapor. It has a spatial resolution of 60m.
+
+In a quick view of the image, we can find that half of the image is blank, we can see that the pixels with high values are the cloud coverage on the top, and the contrast of everything else is quite low. This is within expectation because `B09` is by definition sensitive to the water vapors. However if one would like to have a more detailed view of the ground pixels, one can also make the plot with the option `robust=True`:
+
+~~~
+raster_ams_b9.plot(robust=True)
+~~~
+{: .language-python}
+<img src="../fig/E06-02-overview-plot-B09-robust.png" title="Raster plot with rioxarray using the robust setting" alt="Raster plot with robust setting" width="612" style="display: block; margin: auto;" />
+
+Now the color limit is set in a way fitting most of the values in the image. We have a better view of the ground pixels.
+
+Another information that we're interested in is the CRS, and it can be accessed with `.rio.crs`. We introduced the concept of a CRS in [an earlier
+episode](https://carpentries-incubator.github.io/geospatial-python/03-crs/index.html).
+
+Now we will see how features of the CRS appear in our data file and what
+meanings they have.
+
+## View Raster Coordinate Reference System (CRS) in Python
+We can view the CRS string associated with our Python object using the `crs`
```

Comment:
```suggestion
Another information that we're interested in is the CRS, and it can be accessed with `.rio.crs`. We introduced the concept of a CRS in [an earlier
episode]({{ page.root }}{% link _episodes/03-crs.md %}).
Now we will see how features of the CRS appear in our data file and what
meanings they have. We can view the CRS string associated with our Python object using the `crs`
```